<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1752</width>
    <height>1150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Load data</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Define Model</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>551</width>
               <height>111</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="rOILabel">
                <property name="text">
                 <string>ROI</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsMapLayerComboBox" name="roiLayer">
                <property name="enabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="heightLabel">
                <property name="text">
                 <string>Height</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="heightSpinBox">
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="depthLabel">
                <property name="text">
                 <string>Depth</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="depthSpinBox">
                <property name="minimum">
                 <number>-10000</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
                <property name="value">
                 <number>-3000</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="dTMLabel">
                <property name="text">
                 <string>DTM</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QgsMapLayerComboBox" name="DtmLayer"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Stratigraphy</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>551</width>
               <height>184</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="basalContactsLabel">
                <property name="text">
                 <string>Basal Contacts</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsMapLayerComboBox" name="basalContactsLayer"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Unit Name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsFieldComboBox" name="unitNameField"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="structuralDataLabel">
                <property name="text">
                 <string>Structural Data</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QgsMapLayerComboBox" name="structuralDataLayer"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Format</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="orientationType">
                <item>
                 <property name="text">
                  <string>Strike/Dip</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dip Direction/Dip</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="orientationLabel">
                <property name="text">
                 <string>Strike</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QgsFieldComboBox" name="orientationField"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Dip</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QgsFieldComboBox" name="dipField"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Unit Name</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QgsFieldComboBox" name="structuralDataUnitName"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Faults</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>551</width>
               <height>128</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="faultTracesLabel">
                <property name="text">
                 <string>Fault Traces</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QgsMapLayerComboBox" name="faultTraceLayer"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Fault Name</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QgsFieldComboBox" name="faultNameField"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Dip</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QgsFieldComboBox" name="faultDipField">
                <property name="allowEmptyFieldName">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Displacement</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QgsFieldComboBox" name="faultDisplacementField">
                <property name="allowEmptyFieldName">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Model Setup</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>591</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Stratigraphic Column</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>581</width>
             <height>281</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="stratigraphicColumnContainer">
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Fault Properties</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>571</width>
             <height>281</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <widget class="QComboBox" name="faultSelection"/>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Displacement</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Dip</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Center</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_19">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Major Axis</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Minor Axis</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Intermediate Axis</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Add Elipse to Map</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Displacement Profile</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Model</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>591</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_15">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>19</y>
             <width>571</width>
             <height>611</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QPushButton" name="initModel">
              <property name="text">
               <string>Initialise Model</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QListWidget" name="listWidget"/>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>GroupBox</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Export Model</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Export Model</string>
          </property>
          <widget class="QWidget" name="formLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>19</y>
             <width>561</width>
             <height>177</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>File Format</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>vtk</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>gOcad</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>geoh5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>omf</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Stratigraphic Surfaces</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Fault Surfaces</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Block Model</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Stratigraphy Data</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Fault Data</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="checkBox_5">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="saveButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Interogate Model</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>561</width>
               <height>291</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_6">
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Model contacts</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Add to Project</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
